benchmarks/segmentation_layers.py

```diff
@@ -135,7 +135,7 @@ def _batch_norm_nt():
 def max_pool2d_tensor_iter(self, module):
     def _max_pool2d_tensor_iter():
         for t in self.inputs:
-            module(t.unsqueeze(0)).squeeze(0)
+            res = module(t.unsqueeze(0)).squeeze(0)
```

Comment:
why not accumulate in a list? I'd imagine the garbage collector could still get rid of this the next iteration around since it's not referenced anymore.

Reply:
hmm. fair enough. let me try and see if behavior changes

Reply:
Didn't show any effect yet so removing these changes for now.